.github/workflows/go.yml

```diff
@@ -65,8 +65,15 @@ jobs:
         with:
           go-version: "${{env.GO_VERSION}}"
 
-      - name: Build Docker image
-        run: make update-devel-image
+      - name: Build Forwarder
+        run: go build -o forwarder ./cmd/forwarder
 
       - name: Run e2e test
-        run: cd e2e && ./run.sh
+        run: cd e2e && make test
```

Comment:
make -C e2e

This works as test is the 1st target, you can add `all: test` to make it more explicit in the Makefile.